src/main/kotlin/no/nav/pam/feed/ad/SearchClient.kt

```diff
@@ -27,15 +28,21 @@ fun Route.feed(searchApiHost: String, httpClient: HttpClient) {
     val url = "$searchApiHost/public-feed/ad/_search"
     log.info("Using search API host: ${searchApiHost}")
 
-    get("/api/v1/ads") {
+    try {
+        get("/api/v1/ads") {
+            val subject = call.principal<JWTPrincipal>()?.payload?.subject ?: "?"
+            MDC.put("U", subject)
```

Comment:
Kan hende det burde håndteres annerledes med tanke på corutines, kan hende man mister mdc noen steder

https://github.com/MicroUtils/kotlin-logging/wiki#mdc-and-threadscoroutines